app/controllers/users_controller.rb

```diff
@@ -138,17 +139,21 @@ def update
     return error!(401, 'Wrong user') unless current_user == user
 
     # Finagling things into place
-    changes[:cover_image] = changes[:cover_image_url] if changes[:cover_image_url] =~ /^data:/
-    changes[:password] = changes[:new_password] if changes[:new_password].present?
+    changes[:cover_image] =
+      changes[:cover_image_url] if changes[:cover_image_url] =~ /^data:/
+    changes[:password] =
+      changes[:new_password] if changes[:new_password].present?
     changes[:name] = changes[:new_username] if changes[:new_username].present?
     changes[:star_rating] = (changes[:rating_type] == 'advanced')
-    [:new_password, :new_username, :rating_type, :cover_image_url].each do |key|
+    %i(new_password new_username rating_type cover_image_url).each do |key|
       changes.delete(key)
     end
 
-    changes = changes.permit(:about, :location, :waifu, :website, :waifu_or_husbando,
-                   :bio, :waifu_char_id, :email, :cover_image, :sfw_filter,
-                   :title_language_preference, :password, :name, :star_rating)
+    changes = changes.permit(:about, :location, :waifu, :website,
+                             :waifu_or_husbando, :bio, :waifu_char_id, :email,
+                             :cover_image, :sfw_filter,
+                             :title_language_preference, :password, :name,
+                             :star_rating)
```

Comment:
Changing order might help, if that doesn't help enough, you might be able to realign somehow or maybe use `*%i[about location waifu website]` (a splat operator over a symbol-array literal) — I've never actually tried that but it _should_ theoretically work